rollup.config.js

```diff
@@ -3,22 +3,30 @@ import commonjs from 'rollup-plugin-commonjs'
 import nodeResolve from 'rollup-plugin-node-resolve'
```

Comment:
Pretty much all the changes in this file are to make sure that `process.env.NODE_ENV === 'production'` gets swapped only in the UMD builds.